app/assets/stylesheets/email.scss

```diff
@@ -337,7 +337,7 @@ ul {
 @media only screen and(max-width: 600px) {
   a.btn {
     background-image: none !important;
-    display: block !important;
+    display: inline-block !important;
```

Comment:
The hound config is broken 😞  Commit the version of rubocop.yml in my PR to fix it